src/components/footer/Footer.tsx

```diff
@@ -1,65 +1,81 @@
 import styles from "./footer.module.scss";
-import transitionImg from "../../assets/img/layered-waves-haikei (2).png";
+import transitionImgDarkTheme from "../../assets/img/layered-waves-haikei dark-theme.png";
+import transitionImgLightTheme from "../../assets/img/layered-waves-haikei light-theme.png";
 import FacebookIcon from "../ui/icon/socialMedia/FacebookIcon";
 import InstagramIcon from "../ui/icon/socialMedia/InstagramIcon";
 import TwitterIcon from "../ui/icon/socialMedia/TwitterIcon";
 import TelegramIcon from "../ui/icon/socialMedia/TelegramIcon";
 import { useTranslation } from "react-i18next";
+import classNames from "classnames";
+import ThemeSwitcher from "../ui/themeSwitcher/ThemeSwitcher";
 
-const Footer = () => {
+interface FooterProp {
+  theme: string;
+  setTheme: (theme:string) => void
+}
+
+const Footer = ({ theme,setTheme }: FooterProp) => {
   const { t, i18n } = useTranslation();
-  const languages = ["ua", "en","es"];
+  const languages = ["ua", "en", "es"];
 
   return (
     <div className={styles.footerContainer}>
-      <img src={transitionImg} alt="transitionImage" />
-      <div className={styles.mainContainer}>
+      <img src={theme ? transitionImgLightTheme : transitionImgDarkTheme} alt="transitionImage" />
```

Comment:
what if will be more than 2 themes?
what if you will have theme prop, but it should be the dark theme?